react-app/src/utils/crumbs.test.ts

```diff
@@ -0,0 +1,114 @@
+// crumbs.test.ts
+import { describe, it, expect} from 'vitest';
+import { Authority } from 'shared-types/authority';
+import { Action } from 'shared-types/actions'; 
+import {
+  getDashboardTabForAuthority,
+  detailsAndActionsCrumbs,
+  dashboardCrumb,
+  detailsCrumb,
+  actionCrumb,
+} from './crumbs';
+
+
+
+describe('crumbs', () => {
+    describe('getDashboardTabForAuthority', () => {
```

Comment:
1. I don't know if we need description for the file name. Your other test files don't have them
2. There seems to be some formatting issues. Do you have Prettier installed?

Reply:
No, it wasn't needed and I installed prettier.